internal/uniqid/uniqid.go

```diff
@@ -0,0 +1,202 @@
+package uniqid
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/google/uuid"
+)
+
+// DirLocation represents tokens to indicate where uniqid file should be
+// located.
+type DirLocation int
+
+// DirLocation enums.
+const (
+	InHome DirLocation = iota
+	InTmp
```

Comment:
I THINK this means "based on home dir" and "based on tmp dir". Would be good to have some comments here.